Apps/WebClient/src/ClientApp/src/components/shared/HgCardButtonComponent.vue

```diff
@@ -91,5 +115,8 @@ export default class HgCardButtonComponent extends Vue {
         outline-style: solid;
         outline-color: rgba(86, 86, 86, 0.25);
     }
+    .chevron-icon {
+        color: $primary;
+    }
```

Comment:
Nit: there should be a newline above this rule.

Reply:
Changed.